data-migrations/nuke-db.sh

```diff
@@ -1,11 +1,31 @@
-# Load the Dotenv file
-. .env
 
-# Run the migration using the env variable defined in the dotenv file
-if [ -z "${MYSQL_PASSWORD}" ]; then
-  MYSQL_ARGS="-h ${MYSQL_HOST} -P ${MYSQL_PORT} -u ${MYSQL_USER}"
+if [ ! -f /tmp/foo.txt ]; then
```

Comment:
@briri  is "/tmp/foo.txt" correct?

Reply:
lol. copy/paste.

Will update these and the changelog!